mods/common/chrome/settings-display.yaml

```diff
@@ -359,3 +348,128 @@ Container@DISPLAY_PANEL:
 									Font: Tiny
 									Text: Display and OpenGL changes require restart
 									Align: Center
+				Background@SECTION_HEADER:
+					X: 5
+					Width: PARENT_RIGHT - 24 - 10
+					Height: 13
+					Background: separator
+					ClickThrough: True
+					Children:
+						Label@LABEL:
+							Width: PARENT_RIGHT
+							Height: PARENT_BOTTOM
+							Font: TinyBold
+							Align: Center
+							Text: Accessibility
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 25
+					Children:
+						Container@PLAYER_RELATIONSHIP_COLORS_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Checkbox@PLAYER_RELATIONSHIP_COLORS_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Player Relationship Colors
```

Comment:
You are re-using the same name for something different which irritated me.

Reply:
the current name is Player Relationship Colors, the stance colors is a leftover that I am not changing as I don't want to mess with peoples settings (we don't have a settings update rule)